tooling/hibernate-maven-plugin/hibernate-maven-plugin.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
     intTestRuntimeOnly 'org.apache.maven:maven-compat:3.9.11'
     intTestRuntimeOnly 'org.apache.maven.resolver:maven-resolver-transport-http:1.9.24'
     intTestRuntimeOnly 'org.apache.maven.resolver:maven-resolver-connector-basic:1.9.24'
+
+    intTestRuntimeOnly 'org.crac:crac:1.5.0'
```

Comment:
let's use one of the existing dependencies for other (non-core) modules e.g. `geolatte-geom`